src/integrations/spatiallm/SpatialLMIntegration.ts

```
@@ -0,0 +1,101 @@
+      })),
+      config: {
+        confidenceThreshold: this.config.confidenceThreshold,
+        maxObjects: this.config.maxObjects,
+        minObjectSize: this.config.minObjectSize,
+        maxObjectSize: this.config.maxObjectSize,
+        pointCloudDensity: this.config.pointCloudDensity
+      }
+    };
+  }
+
+  private convertFromSpatialLMFormat(data: any): any {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)